life_expectancy/cleaning.py

```diff
@@ -0,0 +1,188 @@
+import re
+import os
+import argparse
+from typing import TypeVar, Optional
+import pandas as pd
+
+Choosable = TypeVar("Choosable", str, int, float, bool, object)
+COMBINED_COLUMN = 0  # Index of the combined column
+
+
+class RegionNotFoundError(ValueError):
```

Comment:
I see the value of this function! Not sure what happens when the error is raised, for example, would expect it to be logged using a logger, or at least printed

Reply:
I created a custom exception for when the user enters a region that is not valid. So, when the program reaches the function that filters the dataframe by region, if the dataframe is empty, I force the code to raise a RegionNotFoundError (a custom exception). Here is a print of what happens when I type an invalid region. I did this inspired by the video "7 Python Code Smells "  of the first module [lp-foundations](https://github.com/MAlexandraOliveira/lp-foundations/tree/main)/[01_clean_code](https://github.com/MAlexandraOliveira/lp-foundations/tree/main/01_clean_code)/[1_long_code_is_not_good_code](https://github.com/MAlexandraOliveira/lp-foundations/tree/main/01_clean_code/1_long_code_is_not_good_code)
/
<img width="638" alt="image" src="https://github.com/user-attachments/assets/8654d6c7-2ec1-4cec-985a-8f9baea6634e">

Reply:
Makes sense now, I did not see that it will print something, thanks for the clarification.